test/e2e/fqdn_dns_cache_test.go

```diff
@@ -0,0 +1,459 @@
+package e2e
+
+import (
```

Comment:
Please reorganize input into groups: 1. go native imports 2. external libraries 2. antrea packages. Refer to any other go file in Antrea on how those are organized